drodrpg/Texts/MIDs.h

```diff
@@ -1409,6 +1409,7 @@ enum MID_CONSTANT {
   MID_BehaviorAttackInFront = 1526,
   MID_BehaviorAttackAdjacent = 1527,
   MID_BehaviorBeamAttack = 1528,
+  MID_BehaviorBeamBlock = 1748, //New.
```

Comment:
When new MIDs are created, we need to include a text hack into CDbBase::GetMessageText(), so that a user who just installs the .exe as a patch without a new main .dat file will be able to see the text.

Same for any other new MIDs that get added.